src/config/ethereum/common.ts

```diff
@@ -2,6 +2,7 @@ import { AppConfig } from 'config/types';
 
 export const commonConfig: AppConfig = {
   mode: 'development',
+  productionHostName: 'carbondefi.xyz',
```

Comment:
I think it should be part of the external link and not here, since the goal is to use it when the app is **not** carbon

Reply:
There is a distinction to be made here though:

isProduction -> used for when the app is in Production, to hide the Debug menu. Therefore, any partner wanting to fork the code should be able to change what their production hostName is.

isNativeApp -> used to check whether the production app is carbondefi.xyz or not. If it is, then show the footer. Therefore, the native app hostname should be hardcoded vs easily changeable by changing the config